### PR TITLE
TELCODOCS#1626: Recovering VGs from previous LVM Storage installation

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s): 4.15

Issue: [TELCODOCS-1626](https://issues.redhat.com/browse/TELCODOCS-1626)

Link to docs preview:
https://69629--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-creating-lvms-cluster_logical-volume-manager-storage

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@suleymanakbas91 
Could you please review this PR?
Thanks.

**Comment by sr1kar99** [comment]
@mmakwana30 
Could you please review this PR?
Thanks.

File: modules/lvms-creating-logical-volume-manager-cluster.adoc [hunk 1]
```diff
@@ -79,12 +79,14 @@ spec:
...
-<6> Optional: To control what worker nodes the `LVMCluster` CR is applied to, specify a set of node selector labels.
+<5> Optional. To control or restrict the volume group to your preferred devices, you can manually specify the local paths of the devices in the `deviceSelector` section of the `LVMCluster` YAML. The `paths` section refers to devices the `LVMCluster` adds, which means those paths must exist. The `optionalPaths` section refers to devices the `LVMCluster` might add. You must specify at least one of `paths` or `optionalPaths` when specifying the `deviceSelector` section. If you specify `paths`, it is not mandatory to specify `optionalPaths`. If you specify `optionalPaths`, it is not mandatory to specify `paths` but at least one optional path must be present on the node. If you do not specify any paths, then the `LVMCluster` adds the unused devices on the node. After a device is added to the `LVMCluster`, it cannot be removed.
+<6> Optional: To force wipe the selected disks, set `forceWipeDevicesAndDestroyAllData` to true. This parameter is set to `false` by default.
```
Comment (cbippley) [inline modules/lvms-creating-logical-volume-manager-cluster.adoc:88]:
```suggestion
<6> Optional: To force wipe the selected disks, set `forceWipeDevicesAndDestroyAllData` to `true`. This parameter is set to `false` by default.
```

**Review by cbippley — COMMENTED** [review]
One nit, but LGTM!

File: modules/lvms-creating-logical-volume-manager-cluster.adoc (hunk 1 above)
Comment (sr1kar99) [reply modules/lvms-creating-logical-volume-manager-cluster.adoc]: Updated as per the suggestion.

**Comment by dfitzmau** [comment]
Hi @sr1kar99 .This PR looks like it was already reviewed. Needs a `peer-review-done` label?

**Comment by sr1kar99** [comment]
> Hi @sr1kar99 .This PR looks like it was already reviewed. Needs a `peer-review-done` label?

Hi @dfitzmau 
Yes. Could you please add the label?

**Comment by dfitzmau** [comment]
Hi @cbippley . I think you covered the peer review for this PR? If so, is it good for a `peer-review-done` label. Just asking as the PR is in the review queue.

**Comment by nalhadef** [comment]
Hi, @sr1kar99 and @dfitzmau. I'm going to finish the review now I started yesterday.

**Comment by nalhadef** [comment]
All done.

/lgtm

File: modules/lvms-creating-logical-volume-manager-cluster.adoc [hunk 2]
```diff
@@ -79,12 +79,14 @@ spec:
 If you add or remove a `deviceClass`, then the update reflects in the cluster only after deleting and recreating the `TopoLVM-Node` pod.
 Configure the local device paths of the disks as an array of values in the `deviceSelector` field.
 When configuring multiple device classes, you must specify the device path for each device.
-<2> Set `fstype` to `ext4` or `xfs`. By default, it is set to `xfs` if the setting is not specified.
-<3> Mandatory: The `LVMCluster` resource must contain a single default storage class. Set `default: false` for secondary device storage classes.
+<2> Specify a name for the LVM volume group (VG). To recover VGs from the previous {lvms} installation, you must set this field to the name of the VG that you want to recover. 
+You can only recover the VGs but not the logical volume associated with the VG.
+<3> Set `fstype` to `ext4` or `xfs`. By default, it is set to `xfs` if the setting is not specified.
```
[the remark this answers is not in the document]
Comment (jakobmoellerdev) [reply modules/lvms-creating-logical-volume-manager-cluster.adoc:84]: This is personal / deployment preference. By default xfs is chosen for the user if the user does not mind / care. ext4 is useful for certain customers but these are dependant on the workload. 
TLDR: If the customer knows the FS is relevant for them, they will know what to choose here.
